src/libs/Navigation/OnyxTabNavigator.tsx

```diff
@@ -74,6 +78,7 @@ function OnyxTabNavigator({
     screenListeners,
     shouldShowLabelWhenInactive = true,
     disableSwipe = false,
+    lazyLoadScreens = false,
```

Comment:
```suggestion
    lazyLoadEnabled = false,
```
I think we can name it with a more boolean way

Reply:
yeah good idea, thank you! Already updated the PR ✅